### PR TITLE
update localization for Serbia

### PR DESCRIPTION
**Pull request by dnfesenk**

### CODE REVIEW

**Comment by vol1ura** [comment]
Hello, Denis! Thank you for your request and translations. Can you make resolve conflicts here in rs.yml ? I've added english translations, but you can apply yours in serbian language.

**Comment by dnfesenk** [comment]
Hello Yuri! I have updated the localization, but it must first be reviewed by a native speaker; I will let you know when that happens. Until then, please do not merge. Thank you.

**Review by vol1ura — APPROVED** [review]

**Comment by dnfesenk** [comment]
@vol1ura now you can merge it, thank you.

**Pull request merged**
